### ISSUE TITLE
[Release v0.4.0] Prepare it

### ISSUE THREAD

**Issue by dpordomingo** [bblfsh/dashboard#45]
required by https://github.com/src-d/backlog/issues/1074

- [x] ~~Update [`main.VERSION`](https://github.com/bblfsh/dashboard/blob/master/server/cmd/bblfsh-dashboard/main.go#L20) to `v0.4.0`~~ => we moved to tagging by Makefile and don't keep version in code (same as other src-d projects do)
- [x] Release the latest version once the subtasks of https://github.com/src-d/backlog/issues/1074 are checked
- [x] Ensure the `latest` tag at dockerhub points to the new `v0.4.0` image tag https://github.com/bblfsh/dashboard/issues/65 (blocked by ~~https://github.com/src-d/ci/pull/44~~ https://github.com/src-d/ci/pull/47)
- ~~Deploy publicly~~

/cc @marnovo

**Comment by dpordomingo** [comment]
@marnovo let mi know if there is something missing

**Comment by dpordomingo** [comment]
I droped the "Deploy publicly" subtask because Infrastructure will not work on it during [this current print](https://github.com/src-d/backlog/milestone/54)

**Comment by dpordomingo** [comment]
The PR https://github.com/bblfsh/dashboard/pull/48 contains the «load from gist feature» I still need tests and the url can not be shared yet. I hope to complete both in 1~2 working days

**Comment by eiso** [comment]
Closing this issue. I believe it's done.

**Comment by eiso** [comment]
Ah sorry, I looked too quickly. I see the unchecked ones now. My apologies.

**Comment by dpordomingo** [comment]
superseded by https://github.com/src-d/backlog/issues/1074

**Comment by dpordomingo** [comment]
@smacker I updated the issue desc to match our current requirements
(I feel the _"Update main.VERSION to v1.0.0"_ subtask should be done in a separated PR to match to the new release that will be created after this one has been merged)

**Comment by bzz** [comment]
@dpordomingo could you update issue plz a bit:
 - it should be v0.4 in the title
 - lets strike-through ~Deploy publicly~ as it's going to be taken care separately

**Comment by dpordomingo** [comment]
@mcuadros @marnovo could you please give permissions to all Applications team members? All of us should be able to merge PRs and edit issues

**Comment by bzz** [comment]
I think the only thing left is to merge src-d/ci#44 and then tag the release, that is going to be pushed to DockerHub! Let's try to do that asap

**Comment by smacker** [comment]
sadly no. `make packages` is broken now and we need this command to build binary which we wrap in docker container. There is a PR with fix, but I'll rework it (move from bblfsh-dashboard to ci)

**Comment by dpordomingo** [comment]
Yes, the `CGO_ENABLED` in`src-d/co/Makefile.main should be configurable from the outside, keeping the current `0` value as the default, and setting it as `1` to build the Dashboard.
Ping me when the PR is ready and I'll review it :)

**Comment by smacker** [comment]
Done https://hub.docker.com/r/bblfsh/dashboard/tags/
